scripts/run_e2e_tests.py

```diff
@@ -164,6 +163,48 @@
 # the oppia web server.
 SUBPROCESSES = []
 
+RERUN_POLICIES = {
+    'accessibility': 'never',
+    'additionaleditorfeatures': 'known flakes',
+    'additionalplayerfeatures': 'always',
+    'adminpage': 'never',
+    'classroompage': 'never',
+    'classroompagefileuploadfeatures': 'never',
+    'collections': 'never',
+    'contributordashboard': 'always',
+    'coreeditorandplayerfeatures': 'known flakes',
+    'creatordashboard': 'always',
+    'embedding': 'known flakes',
+    'explorationfeedbacktab': 'never',
+    'explorationhistorytab': 'always',
+    'explorationstatisticstab': 'never',
+    'explorationtranslationtab': 'never',
+    'extensions': 'known flakes',
+    'fileuploadextensions': 'never',
+    'fileuploadfeatures': 'known flakes',
+    'learner': 'known flakes',
+    'learnerdashboard': 'known flakes',
+    'library': 'known flakes',
+    'navigation': 'never',
+    'playvoiceovers': 'never',
+    'preferences': 'known flakes',
+    'profilefeatures': 'never',
+    'profilemenu': 'never',
+    'publication': 'never',
+    'skilleditor': 'known flakes',
+    'subscriptions': 'never',
+    'topicandstoryeditor': 'always',
+    'topicandstoryeditorfileuploadfeatures': 'never',
+    'topicandstoryviewer': 'known flakes',
+    'topicsandskillsdashboard': 'always',
+    'users': 'known flakes',
+    'wipeout': 'known flakes',
+    # The following are not real suites; they're only for testing.
+    'never': 'never',
+    'known_flakes': 'known flakes',
+    'always': 'always',
```

Comment:
Lets not add unwanted code here for testing purposes, can we swap this constant in the test file and test?

Reply:
done